server/src/models/user.model.ts

```diff
@@ -0,0 +1,7 @@
+export type User = {
```

Comment:
I am adding Telegram type's, let's wait for a while

Reply:
Good